### PR TITLE
LPS-110054 Add missing properties

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by leoadb** [comment]
Just started reviewing :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_5_0/UpgradeDDMStructure.java [hunk 1]
```diff
@@ -591,10 +592,15 @@ private void _upgradeStructureLayoutDefinition() throws Exception {
 
 			try (ResultSet rs = ps1.executeQuery()) {
 				while (rs.next()) {
+					String definition = rs.getString("definition");
+
+					if (Validator.isNotNull(rs.getLong("parentStructureId"))) {
+						definition = _upgradeDDMFormLayoutDefinition(
+							definition, rs.getLong("structureId"));
```
Comment (leoadb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_5_0/UpgradeDDMStructure.java:603]: @marcelabc is this right? Shouldn't be **rs.getLong("parentStructureId")** ?

**Comment by leoadb** [comment]
ci:stop

**Comment by leoadb** [comment]
sent to https://github.com/leoadb/liferay-portal/pull/796

**Pull request closed without merging**